app_backend/database.py

```diff
@@ -9,11 +9,12 @@ def __init__(self, username: str, password: str):
 
     def connect(self):
```

Comment:
Your changes to these files look good to me (the comments were helpful but I had a few questions that I mentioned below)